content/ai/concepts/neural-networks/terms/convolutional-neural-networks/convolutional-neural-networks.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Convolutional Neural Networks'
+Description: 'Convolutional Neural Networks are a type of neural network that are primarily used for image classification.'
+Subjects:
+  - 'Computer Science'
+Subjects:
+  - 'Deep Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Networks'
+  - 'CNN'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/machine-learning-engineer'
+---
+
+# What is a CNN?
+
+CNNs are a **type of neural network** that are primarily used for image classification. They are also used for other tasks such as object detection, image segmentation, and natural language processing.
```

Comment:
```suggestion
**Convolutional Neural Networks** (CNNs) are a type of neural network that are primarily used for image classification. They are also used for other tasks such as object detection, image segmentation, and natural language processing.
```
Spell out an acronym the first time its used. And we bold the first appearance of the entry term.